### DISCUSSION TITLE
Card is compatible with Titan Accelerator IIe -- in fast mode!

### DISCUSSION THREAD

**Discussion by buserror** [tjboldt/ProDOS-ROM-Drive#17] [General]
Not a bug, but I don't know how to report that anywhere else!

I just got a 3.5Mhz Accelerator IIe card, and I tried first in 'slow mode' where you configure the card to slow down accesses to the slot you want, as well as slow down the access to the IO memory.
It works fine of course, but then I tried to disable the slowdown on both card and IO memory access and ... it still works great!

So well done ;-)

**Comment by tjboldt** [comment]
Thanks. I have created a "Discussions" section and moved your *issue* here.
